_includes/js/queries.md

```diff
@@ -391,3 +424,151 @@ mainQuery.find()
     // There was an error.
   });
 </code></pre>
+
+## Aggregate
+
+Queries can be made using aggregates, allowing you to retrieve objects over a set of input values. The results will not be `Parse.Object`s since you will be aggregating your own fields
+
+* Note: MasterKey is Required. Parse Server 2.7.1+
```

Comment:
Same thing here, we should move this statement directly below the title of `Aggregate`. Should make sure it's similar to the others as well. We can keep the master key requirement in there though, makes sense to have that as well.

Reply:
Also I'm not set in stone on this one. Using an `*` marks it with a blue + on the docs, and that might be good as is, but location should be standardized below the title.